src/server.js

```diff
@@ -210,36 +146,6 @@ export class Server {
         return this._sendNormalRequest(url);
```

Comment:
Looks like it's the only place we're using `_sendNormalRequest` in this file and it duplicates `_sendNormalRequest` in call_builder.js. Can we follow a new convention and create `friendbot_call_builder.js`?